src/client/components/collection/Badges.js

```diff
@@ -25,8 +26,27 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function Badges({ views, likes }) {
+function Badges({ views, likes, name }) {
   const classes = useStyles();
+  const [pageLocation, setPageLocation] = useState(null);
+
+  useEffect(() => {
+    setPageLocation(window?.location.href);
+  });
+
+  const shareOnTwitter = () => {
+    const textToShare = `${name} ${pageLocation}`;
+    openInNewTab(`https://twitter.com/intent/tweet?text=${textToShare}`);
+  };
+
+  const shareOnFacebook = () => {
+    const link = pageLocation;
+    openInNewTab(`https://www.facebook.com/sharer/sharer.php?u=${link}`);
+  };
+
+  const subject = name;
+  const message = pageLocation;
```

Comment:
Maybe we can have the content coming from the `t` JSON files. Something simple like: "Check out this collection on Graasp." And then maybe appending the collection description, if any.